tests/integration-tests/tests/dns/test_dns.py

```diff
@@ -23,9 +23,7 @@
 @pytest.mark.regions(["eu-west-2"])
 @pytest.mark.instances(["c5.xlarge"])
 @pytest.mark.schedulers(["slurm"])
-def test_hit_no_cluster_dns_mpi(
-    scheduler, region, os, instance, pcluster_config_reader, clusters_factory, architecture, test_datadir
-):
+def test_hit_no_cluster_dns_mpi(scheduler, region, instance, pcluster_config_reader, clusters_factory, test_datadir):
```

Comment:
we can't remove os from the fixture otherwise we broke the parametrization on the default dimension. we can move to usefixtures annotation at most.

Reply:
Fixed